pandas/tests/copy_view/test_array.py

```diff
@@ -119,3 +119,57 @@ def test_ravel_read_only(using_copy_on_write, order):
     if using_copy_on_write:
         assert arr.flags.writeable is False
     assert np.shares_memory(get_array(ser), arr)
+
+
+def test_series_array_ea_dtypes(using_copy_on_write):
+    ser = Series([1, 2, 3], dtype="Int64")
+    arr = np.asarray(ser, dtype="int64")
+    assert np.shares_memory(arr, get_array(ser))
+    if using_copy_on_write:
+        assert arr.flags.writeable is False
+    else:
+        assert arr.flags.writeable is True
+
+    arr = np.asarray(ser)
+    assert not np.shares_memory(arr, get_array(ser))
+    assert arr.flags.writeable is True
+
+
+def test_dataframe_array_ea_dtypes(using_copy_on_write):
+    df = DataFrame({"a": [1, 2, 3]}, dtype="Int64")
+    arr = np.asarray(df, dtype="int64")
+    # TODO: This should be able to share memory, but we are roundtripping
+    # through object
+    assert not np.shares_memory(arr, get_array(df, "a"))
+    assert arr.flags.writeable is True
+
+    arr = np.asarray(df)
+    if using_copy_on_write:
+        # TODO(CoW): This should be True
```

Comment:
Not this one, because without specifying `dtype="int64"` we create an object dtype array?

Reply:
Yes exactly, this triggers a copy and hence the array should be writeable?